.github/workflows/docker-publish.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       run: |
         ARCH=$(cut -d "/" -f2 <<< ${{ matrix.docker-platform }})
         DOCKER_BUILDKIT=1 docker buildx build --platform=$ARCH --load \
```

Comment:
Doesn't it matter --platform=$ARCH or --platform=${{ matrix.docker-platform }}?

Reply:
Thanks - updated!